c-vendor/vendor.sh

```diff
@@ -5,10 +5,7 @@ set -e
 # XXX: provide a nice declarative format
 # XXX2: reuse vendor.json and write some python3 code?
 if [ ! -d ./squashfuse ]; then
-    git clone https://github.com/snapcore/squashfuse
+    git clone https://github.com/vasi/squashfuse
```

Comment:
so to be clear, the only reason we used this fork was because we needed to be able to import squashfuse as a Go package via govendor, but now that we have this c-vendor dir we can just checkout upstream and build it directly ?

Reply:
Correct